maintenance/wikia/updateMainPageFilterInMetricProcessor.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+/**
+ * @ingroup Maintenance
+ */
+
+require_once( dirname( __FILE__ ) . '../../Maintenance.php' );
```

Comment:
What do you think about moving portability-related scripts to a subdir of maintenance/wikia ?